### PR TITLE
Add new CSV validator script, refs #13475

### PR DESCRIPTION
**Pull request by sbreker**
Add new validator for the scriptOfDescription column. Validate using sfCultureInfo->getScripts().

Includes new phpunit test for the new validator.

### CODE REVIEW

File: lib/task/import/validate/csvParentValidator.class.php [hunk 1]
```diff
@@ -162,14 +162,23 @@ public function getTestResult()
                 && $this->parentIdColumnPresent
                 && $this->orphanRowsFound
             ) {
-                $this->testData->setStatusWarn();
-                $this->testData->addResult(sprintf("'source' option not specified. Unable to check parentId values against AtoM's database."));
+                //$this->testData->setStatusInfo();
```
Comment (mcantelon) [inline lib/task/import/validate/csvParentValidator.class.php]: Dead code

File: lib/task/import/validate/csvParentValidator.class.php [hunk 2]
```diff
@@ -162,14 +162,23 @@ public function getTestResult()
...
+                $this->testData->addResult(sprintf('Verifying parentId values against legacyId values in this file.'));
+            }
+
+            if (
+                !empty($this->options['source'])
+                && $this->parentIdColumnPresent
+                && $this->orphanRowsFound
+            ) {
+                //$this->testData->setStatusInfo();
```
Comment (mcantelon) [inline lib/task/import/validate/csvParentValidator.class.php]: Dead code

**Review by mcantelon — APPROVED** [review]
There's a couple lines of dead code but other than that it looks good to me!

**Comment by sbreker** [comment]
Thanks @mcantelon!  Will remove those two lines and merge

File: lib/task/import/validate/csvParentValidator.class.php (hunk 1 above)
Comment (sbreker) [reply lib/task/import/validate/csvParentValidator.class.php]: Removed

File: lib/task/import/validate/csvParentValidator.class.php (hunk 2 above)
Comment (sbreker) [reply lib/task/import/validate/csvParentValidator.class.php]: Removed

**Pull request merged**